File: M241.Server/Data/Models/Room.cs
﻿using System.Text.Json.Serialization;

namespace M241.Server.Data.Models
{
    public class Room
    {
        [JsonPropertyName("id")]
        public int Id { get; set; }
        [JsonPropertyName("name")]
        public string Name { get; set; } = string.Empty;

        [JsonPropertyName("macAddress")]
        public string MACAddress { get; set; } = string.Empty;
        [JsonPropertyName("isBurning")]
        public bool IsBurning { get; set; }
    }
}

File: M241.Frontend/models/Room.ts
class Room {
  id?: number;
  name: string;
  macAddress: string;
  isBurning: boolean;

  constructor(
    id: number,
    name: string,
    macAddress: string,
    isBurning: boolean
  ) {
    this.id = id;
    this.name = name;
    this.macAddress = macAddress;
    this.isBurning = isBurning;
  }
}

export default Room;
